src/rain-report/rain-report.service.ts

```diff
@@ -0,0 +1,22 @@
+import { Injectable } from '@nestjs/common';
+import { RainReport } from '@prisma/client';
+import { PrismaClient } from '@prisma/client';
+
+@Injectable()
+export class RainReportService {
+  /**
+   * Retrieve all `RainReport` records from the database.
+   */
+  async all(): Promise<Partial<RainReport>[]> {
+    // TODO: create a service and inject
+    const database = new PrismaClient();
+    try {
+      return await database.rainReport.findMany({
```

Comment:
The example response provided (see the **Description** of this PR above for details) shows:

* only the `timestamp` and `rain` properties are to be returned
* ordering appears to be by `timestamp` in descending order

The database query has been written to reflect this.